tests/end_to_end/test_target_bigquery.py

```diff
@@ -41,6 +41,8 @@ def setup_method(self):
         self.run_query_tap_postgres = self.e2e.run_query_tap_postgres
         self.run_query_target_bigquery = self.e2e.run_query_target_bigquery
         self.mongodb_con = self.e2e.get_tap_mongodb_connection()
+        self.e2e.setup_target_bigquery()
+        self.e2e.remove_all_state_files()
```

Comment:
can this moved to the `teardown_method` instead?

Reply:
The state files need to be removed before a test runs not after. Eg. if I've run PipelineWise manually for some reason in dev then want to run the tests the state files need to go. Similarly for emptying the BigQuery datasets. 

Furthermore, removing this data on teardown is likely to make debugging failures more difficult (since the tests seems to exit on the first failure).